.github/pull_request_template.md

```diff
@@ -3,7 +3,8 @@
 3. Deploy: [link](https://github.com/)
 4. Done 28.05.2020 / deadline 31.05.2020
 5. Score: 75 / 110
-  - [x] Application is divided to logical modules / layers. Like working with api it's separate module, as well as working with UI rendering, or working with an application state, etc. Better discuss architecture with a mentor before implementing. (40/40)
-  - [x] All the HTML-content is generated by JavaScript (body contains only one tag script inside). (15/30) 
-  - [ ] Application is SPA.(0/20)
-  - [x] Webpack or another bundler should be used. There should be one HTML-file, JS-file, and one CSS-file (or it can be embedded right in the HTML-file). (20/20)
+
+- [x] Application is divided to logical modules / layers. Like working with api it's separate module, as well as working with UI rendering, or working with an application state, etc. Better discuss architecture with a mentor before implementing. (40/40)
+- [x] All the HTML-content is generated by JavaScript (body contains only one tag script inside). (15/30)
+- [ ] Application is SPA.(0/20)

```

Comment:
- [x] All the HTML-content is generated by JavaScript (body contains only one tag script inside). = I've replaced root id into body and got warning that it needs container, add script to header with defer
didnt find any other solutions on that,  without warnings
SPA its one page html responsible for re-render dynamically changed content without requesting data from server in browser so it has to be reloaded
Application is not SPA because i didn't used Router on this module ? I have it on this week's task,  branch module02, or did I miss smth?))